meta-resin-common/recipes-connectivity/networkmanager/networkmanager_%.bbappend

```diff
@@ -14,6 +15,7 @@ PACKAGECONFIG_append = " systemd modemmanager ppp"
 PACKAGES += "${PN}-bash-completion"
 
 FILES_${PN}-bash-completion = "${datadir}/bash-completion"
+RESIN_BOOT_PARTITION_FILES_append = " system-connections/examples/connection.example:/system-connections/examples/connection.example"
```

Comment:
you need to add it to both resin-image.bb and resin-image-flasher.bb

Like:

https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-core/images/resin-image-flasher.bb#L38

and 

https://github.com/resin-os/meta-resin/blob/master/meta-resin-common/recipes-core/images/resin-image.bb#L57